website/blog/2023-01-11-databend-weekly-76.md

```diff
@@ -0,0 +1,155 @@
+
+**Query**
+
+- efficiently memory two level group by in standalone mode ([#9504](https://github.com/datafuselabs/databend/pull/9504))
+
+**Storage**
+
+- support nested type in `read_parquet` ([#9486](https://github.com/datafuselabs/databend/pull/9486))
+- add build options table ([#9502](https://github.com/datafuselabs/databend/pull/9502))
+
+### Code Refactoring :tada:
+
+**\***
+
+- merge new expression ([#9411](https://github.com/datafuselabs/databend/pull/9411))
+- remove and rename crates ([#9481](https://github.com/datafuselabs/databend/pull/9481))
+- bump rust version ([#9540](https://github.com/datafuselabs/databend/pull/9540))
+
+**Expression**
+
+- move negative funtions to binder ([#9484](https://github.com/datafuselabs/databend/pull/9484))
+- use error_to_null() to eval try_cast ([#9545](https://github.com/datafuselabs/databend/pull/9545))
+  
+**Functions**
+
+- replace h3ron to h3o ([#9553](https://github.com/datafuselabs/databend/pull/9553))
+
+**Format**
+
+- extract `AligningStateTextBased` ([#9472](https://github.com/datafuselabs/databend/pull/9472))
+- richer error context ([#9534](https://github.com/datafuselabs/databend/pull/9534))
+
+**Query**
+
+- use ctx to store the function evaluation error ([#9501](https://github.com/datafuselabs/databend/pull/9501))
+- refactor map access to support view read tuple inner ([#9516](https://github.com/datafuselabs/databend/pull/9516))
+
+**Storage**
+
+- bump opendal for streaming read support ([#9503](https://github.com/datafuselabs/databend/pull/9503))
+- refactor bloom index to use vectorized siphash function ([#9542](https://github.com/datafuselabs/databend/pull/9542))
+
+### Bug Fixes :wrench:
+
+**HashTable**
+
+- fix memory leak for unsized hash table ([#9551](https://github.com/datafuselabs/databend/pull/9551))
+
+**Storage**
+
+- fix row group stats collection ([#9537](https://github.com/datafuselabs/databend/pull/9537))
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### Databend Now Works with New Expression
+
+After more than half a year of development and migration, Databend now works entirely with **New Expression** .
+
+New Expression brings a *formal type system* to Databend and supports *type-safe downward casting* . Also, it will be easier to define functions in the new expressions.
+
+We are in the process of resolving a small number of remaining issues and will release a new v0.9 version once it has been fully stabilised.
+
+**Learn More**
+
+- [PR | refactor: Merge new expression](https://github.com/datafuselabs/databend/pull/9411)
+- [Issue | Tracking: issues about new expression](https://github.com/datafuselabs/databend/issues/9480)
+
+## What's Up Next
+
+We're always open to cutting-edge technologies and innovative ideas. You're more than welcome to join the community and bring them to Databend.
+
+####  Support `unnest`
```

Comment:
### UNNEST Function

The UNNEST function takes an array as a parameter, and returns a table containing each element of the array in a row.

If you're interested in becoming a contributor, helping us develop the UNNEST function would be a good start.